# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /Users/vitthal/Desktop/drupal/trunk/modules/taxonomy
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: taxonomy.module
--- taxonomy.module Base (BASE)
+++ taxonomy.module Locally Modified (Based On LOCAL)
@@ -320,6 +320,10 @@
     $status = SAVED_NEW;
   }
 
+/**************************************************************************
+Adjustment by D. Shorthouse to accommodate directionality of term relations
+**************************************************************************/
+/*
   db_query('DELETE FROM {term_relation} WHERE tid1 = %d OR tid2 = %d', $form_values['tid'], $form_values['tid']);
   if (!empty($form_values['relations'])) {
     foreach ($form_values['relations'] as $related_id) {
@@ -328,6 +332,8 @@
       }
     }
   }
+*/
+/*************************************************************************/
 
   db_query('DELETE FROM {term_hierarchy} WHERE tid = %d', $form_values['tid']);
   if (!isset($form_values['parent']) || empty($form_values['parent'])) {
@@ -502,7 +508,13 @@
  *  and provide scalable alternatives.
  */
 function taxonomy_form_alter(&$form, $form_state, $form_id) {
+/*********************************************************************
+  Adjusted by D. Shorthouse
+
+  original:
   if (isset($form['type']) && isset($form['#node']) && (!variable_get('taxonomy_override_selector', FALSE)) && $form['type']['#value'] .'_node_form' == $form_id) {
+*********************************************************************/
+  if (isset($form['type']) && isset($form['#node']) && $form['type']['#value'] .'_node_form' == $form_id) {
     $node = $form['#node'];
 
     if (!isset($node->taxonomy)) {
